gradle/root/coverage.gradle

```diff
@@ -46,4 +28,19 @@ task mockitoCoverage(type: JacocoReport) {
     }
 }
 
-task coverageReport(dependsOn: ["mockitoCoverage"]) {}
+gradle.taskGraph.whenReady {
+    mockitoCoverage.configure {
```

Comment:
Can we move this into the task configuration instead? I know this was required back then, but I think they fixed this issue now and we can move it back into the same task.

Reply:
Hmm, I can try. afterEvaluate doesn't work inside register, but maybe whenReady does.

Reply:
Did you mean this? https://github.com/mockito/mockito/pull/3052/commits/97151b03242ee2ed56618be5fb756058c8189d59 or I misunderstood?

Reply:
Note: I moved it out in https://github.com/mockito/mockito/pull/3052/commits/32894517b02ab0e26a6753acb7de2f6ece69c9b5 because it was running an O(N^2) setup. After each subproject was evaluated, it re-set the trees, but some of the trees were already resolved, and this repeated over and over again.